auth copy.py

```
@@ -0,0 +1,108 @@
+
+def complex_auth_function(user, pass_word, email, phone, address, city, state, zip_code, country, age, gender, preferences):
+    if user and pass_word:
+        if email:
+            if phone:
+                if address:
+                    if city:
+                        if state:
+                            if zip_code:
+                                if country:
+                                    if age > 18:
+                                        if gender:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)